mis_builder/models/mis_report_instance.py

```diff
@@ -592,6 +592,10 @@ def _compute_pivot_date(self):
         help="Search view to customize the filter box in the report widget.",
     )
 
+    wide_display_by_default = fields.Boolean(
```

Comment:
Wasn't the previous behavior to be expanded by default and now we are setting the opposite one?